CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.5.1 (27 Feb, 2018)
```

Comment:
please, remove this change. Changelog is updated during version bump commit, which happens **after** merge